modules/common-attributes.adoc

```diff
@@ -55,7 +55,7 @@ endif::[]
 :osp: Red{nbsp}Hat OpenShift
 :olm-first: Operator Lifecycle Manager (OLM)
 :olm: OLM
-:rhacs-version: 4.5.1
+:rhacs-version: 4.6.0
 :ocp-supported-version: 4.11
```

Comment:
looking at the support matrix, it looks like we are supporting 4.12 at a minimum, instead of 4.11 - https://access.redhat.com/articles/7045053. this is the case for 4.4 and 4.5, too

not sure who decides this